src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-optimizing.mdx

```diff
@@ -619,13 +633,12 @@ The different approaches:
           * Logs generated per cluster
         </Callout>
 
-        It's not surprising that a complex and decentralized system like Kubernetes has the potential to generate a lot of telemetry fast.  There are a few good approaches to managing data ingest in Kubernetes.  These will be very straightforward if you are using observability as code in your K8s deployments.
+        It's not surprising that a complex and decentralized system like Kubernetes has the potential to generate a lot of telemetry fast.  There are a few good approaches to managing data ingest in Kubernetes.  These will be very straightforward if you are using observability as code in your K8s deployments.  We highlly recommend you install this `Kubernetes Data Ingest Analysis` dashboard for making any decisions about reducing K8s ingest.  The dashbaord is available in this [quickstart](https://newrelic.com/instant-observability/infrastructure-integrations-data-analysis/8e31a0ae-81c0-4df0-a119-a0ada9ec16fa).
 
         ### Scrape interval
 
         Depending on your observability objectives you may consider adjusting the scrape interval.  The default is 15s.  Now the Kubernetes cluster explorer only refreshes every 45s.  If your primary use of the K8s data is to support the KCE visualizations you may consider changing your scrape interval to 20s.  That change from 15s to 20s can have a substantial impact.
-
-        ![Update Scrape Interval](images/oma-oe-dg-update-k8s-scrape-interval.png)
+        See [this section](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#scrape-interval) of the New Relic docs for more details.
```

Comment:
@JimHagan, how about a more specific link like this: See more details about [scrape interval](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#scrape-interval) in our Helm documentation.